readme.md

```diff
@@ -708,6 +708,7 @@
 - [Biomedical Information Extraction](https://github.com/caufieldjh/awesome-bioie#readme) - How to extract information from unstructured biomedical data and text.
 - [Web Archiving](https://github.com/iipc/awesome-web-archiving#readme) - An effort to preserve the Web for future generations.
 - [WP-CLI](https://github.com/schlessera/awesome-wp-cli#readme) - Command-line interface for WordPress.
+- [Veganism](https://github.com/sdassow/awesome-veganism#readme) - Resources for a fun and smooth vegan experience.
```

Comment:
`Resources for ....` is not compliant with description requirements.

Reply:
Right. Changed it in 808e99685a01ec9cbe216956fa3c1e1f3a639f56, better?

Reply:
Yes. Thank you.